base/applications/network/arp/arp_msg.mc

```diff
@@ -184,6 +242,9 @@ Language=English
 Language=Polish
   %1!-20s!  %2!-20s!  %3!-10s!
 .
+Language=Romanian
+  %1!-20s!  %2!-20s!  %3!-10s!
+.
```

Comment:
Are these three lines necessary?

Reply:
Yes, because otherwise, the file crashes. The action finishes with an error.